### PR TITLE
update demo script (test only LockBit, update PROTECTED codes)

### PR DESCRIPTION
**Pull request by makk94**
tested on my mac and a windows 2022 VM

### CODE REVIEW

File: shell/demo/demo.ps1 [hunk 1]
```diff
@@ -8,17 +8,19 @@ param(
...
 }
 
 $Tests = [ordered]@{
-    "39de298a-911d-4a3b-aed4-1e8281010a9a" = "Health check"
-    "3ebbda49-738c-4799-a8fb-206630cf609e" = "Will a long running VST be stopped properly?"
-    "2e705bac-a889-4283-9a8e-a12358fa1d09" = "Will your computer quarantine Royal Ransomware?"
-    "b74ad239-2ddd-4b1e-b608-8397a43c7c54" = "Will your computer quarantine a malicious Office document?"
-    "ca9b22be-93d5-4902-95f4-4bc43a817b73" = "Will your computer quarantine Colour-Blind malware?"
+    "39de298a-911d-4a3b-aed4-1e8281010a9a" = "Health Check"
+    "3ebbda49-738c-4799-a8fb-206630cf609e" = "Long Running VST"
```
Comment (privateducky) [inline shell/demo/demo.ps1:19]: let's not include 'long running VST'... I think this test may not exist soon...

**Review by privateducky — CHANGES_REQUESTED** [review]
@makk94 I think you misunderstood me actually...
but it's ok, because we had to do this anyway this week 

1. the demo_account function on Service is the one that needs the emerging threat included
2. these demo scripts should be trimmed down to ONLY include the LockBit (NGAV) test.

sorry for the confusion but at least you're knocking both things out at once!

**Comment by makk94** [comment]
Ohhhh
lol. too many demo things. k, i'll go update the demo account

any reason we're moving back to this demo being a single test instead a list of 5?

**Comment by privateducky** [comment]
> Ohhhh lol. too many demo things. k, i'll go update the demo account
> 
> > 2. these demo scripts should be trimmed down to ONLY include the LockBit (NGAV) test.
> 
> (readme says this is AV not NGAV btw) Screenshot 2023-05-08 at 12 35 07 PM
> 
> any reason we're moving back to this demo being a single test instead a list of 5? (is this a long term decision/ should I rewrite demo to assume single test again)

1. Short term move to 1 test, as we're testing out some new things.
2. We have 2 LockBit tests, one for AV and one for NGAV. The demo should be the latter.

**Review by privateducky — APPROVED** [review]

**Comment by privateducky** [comment]
@makk94 great/clean work. let me know once you've confirmed against staging

**Comment by makk94** [comment]
I tested this against staging already. unless you mean teh full flow (ie. i need to bother patrick to update demo in s3 and then test from download)?

**Pull request merged**